test/mysql.test.js

```diff
@@ -633,6 +633,15 @@ describe('mysql', function() {
               });
             });
       });
+
+      it('filter with case sensitive regex string', function(done) {
```

Comment:
Suggestion: can we make the regex match something for string and regex object? i.e. have results